src/rules/max-expects.ts

```diff
@@ -0,0 +1,89 @@
+import { AST_NODE_TYPES } from '@typescript-eslint/utils';
+import {
+  FunctionExpression,
+  createRule,
+  isExpectCall,
+  isTypeOfJestFnCall,
+} from './utils';
+
+export default createRule({
+  name: __filename,
+  meta: {
+    docs: {
+      category: 'Best Practices',
+      description: 'Enforces a maximum assertion calls in a test body',
+      recommended: false,
+    },
+    messages: {
+      exceededMaxAssertion:
+        'Too many assertion calls ({{ count }}). Maximum allowed is {{ max }}.',
+    },
+    type: 'suggestion',
+    schema: [
+      {
+        type: 'object',
+        properties: {
+          max: {
+            type: 'integer',
+            minimum: 0,
```

Comment:
What do you think about having 1 as the minimum? I can't think of a situation where you'd ever want to disallow _any_ exceptions (even if you were wanting to not use `expect`, there are better rules for enforcing that)

```suggestion
            minimum: 1,
```

Reply:
Fixed in https://github.com/jest-community/eslint-plugin-jest/pull/1166/commits/8bba714d62e447a1680ae4df59017c2fefd55a18. I too think 1 is the appropriate minimum value.
But I can't figure out if something would be needed that would stop the process when a value less than 1 is passed.

Reply:
I didn't understand how validation works. Thanks for explaining it to me!